src/Gui/FreeCADGuiInit.py

```diff
@@ -301,9 +301,9 @@ def _SoGroup_init(self,*args):
 Gui.activateWorkbench("NoneWorkbench")
 
 # Register .py, .FCScript and .FCMacro
-FreeCAD.addImportType("Inventor V2.1 (*.iv)","FreeCADGui")
-FreeCAD.addImportType("VRML V2.0 (*.wrl *.vrml *.wrz *.wrl.gz)","FreeCADGui")
-FreeCAD.addImportType("Python (*.py *.FCMacro *.FCScript)","FreeCADGui")
+FreeCAD.addImportType("Inventor V2.1 (*.iv *.IV)","FreeCADGui")
+FreeCAD.addImportType("VRML V2.0 (*.wrl *.WRL *.vrml *.VRML *.wrz *.WRZ *.wrl.gz *.WRL.GZ)","FreeCADGui")
+FreeCAD.addImportType("Python (*.py *.PY *.FCMacro *.FCScript)","FreeCADGui")

```

Comment:
PY isn't a valid file extension for Python modules.
`touch /tmp/mytest.PY`
Start FreeCAD or Python interpreter:
```
import sys
sys.path.append("/tmp")
import mytest
```
==> 
```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/usr/lib/python3/dist-packages/shiboken2/files.dir/shibokensupport/__feature__.py", line 142, in _import
    return original_import(name, *args, **kwargs)
ModuleNotFoundError: No module named 'mytest'
```

Reply:
Sorry, I didn't know)

Reply:
maybe also add *.fcmacro and *.fcscript?

Reply:
@adrianinsaval ,
Files with these extensions are written only by FreeCAD and only in this form. There's no point in changing it.

Reply:
manually saved scripts from a generic text editor may have just lowercase extensions

Reply:
@adrianinsaval,
Added *.fcmacro *.fcscript.